src/app/projects/page.js

```diff
@@ -137,7 +138,19 @@ export default async function Projects() {
     return projectsResult;
   }, []).flat();
 
+  const isConnected = (project) => {
+    const hasCycleField = project.cycle?.dataType === 'ITERATION' ||  project.cycle?.dataType === 'SINGLE_SELECT'
+    const hasKindField =
+      project.kind?.dataType === 'SINGLE_SELECT' &&
+      ['Pitch', 'Bet'].every(kind => !!project.kind?.options?.find(opt => opt.name === kind))
+    const hasAppetiteField = project.appetite?.dataType === 'SINGLE_SELECT'
+    const isConnected = hasCycleField && hasKindField && hasAppetiteField
+    return isConnected
+  }
+
   const allProjects = [userProjects, orgProjects].flat()
+  const myProjects = allProjects.filter(isConnected)
+  const remaining = allProjects.filter(x=>!myProjects.includes(x))
```

Comment:
Thanks @koukama for this contribution.
Could you have a consistent naming ? `remaining` -> `remainingProjects`